packages/cli/test/unit/execution.lifecycle.test.ts

```diff
@@ -30,138 +30,77 @@ function getDataId(run: IRun, kind: 'binary' | 'json') {
 
 const binaryDataService = mockInstance(BinaryDataService);
 
-describe('on filesystem mode', () => {
-	describe('restoreBinaryDataId()', () => {
-		beforeAll(() => {
-			config.set('binaryDataManager.mode', 'filesystem');
-		});
-
-		afterEach(() => {
-			jest.clearAllMocks();
-		});
-
-		it('should restore if binary data ID is missing execution ID', async () => {
-			const executionId = '999';
-			const incorrectFileId = 'a5c3f1ed-9d59-4155-bc68-9a370b3c51f6';
-			const run = toIRun({
-				binary: {
-					data: { id: `filesystem:${incorrectFileId}` },
-				},
+for (const mode of ['filesystem-v2', 's3'] as const) {
```

Comment:
Obscured by the diff - only change was removal of legacy filesystem mode tests and looping.